docs/src/pages/docs/api.md

```diff
@@ -26,6 +26,7 @@ const {
   enabled,
   initialData,
   initialStale,
+  placeholderData,
```

Comment:
Maybe also add the `isPlaceholderData` to the result?

Reply:
Oh, it should be there now, just forgot to document it.